bochs/bx_debug/lexer.l

```diff
@@ -117,6 +117,8 @@ writemem        { bxlval.sval = strdup(bxtext); return(BX_TOKEN_WRITEMEM); }
 loadmem         { bxlval.sval = strdup(bxtext); return(BX_TOKEN_LOADMEM); }
 setpmem         { bxlval.sval = strdup(bxtext); return(BX_TOKEN_SETPMEM); }
 deref           { bxlval.sval = strdup(bxtext); return(BX_TOKEN_DEREF); }
+telescope       { bxlval.sval = strdup(bxtext); return(BX_TOKEN_TELESCOPE); }
+telescopeset    { bxlval.sval = strdup(bxtext); return(BX_TOKEN_TELESCOPESET); }
```

Comment:
might be also telescope reset shortcut instead of 'telescope 0 0 0 0' ?
also lexer is not forgiving for these 4 params
if you forget one or miss order of these params -> result is not nice and hard to understand what went wrong
I prefer smth like telescope rows=R depth=D data_depth=DD hex=H
but I didn't conclude it yet ... anyway -> at least when telescope is called -> print what is actually did.
like setting 'rows = R and etc
Are where default params here ?
Like hex=TRUE is omitted ?
Or data_depth = max_depth if omitted ?